game/scripts/npc/abilities/juggernaut_omni_slash.txt

```diff
@@ -48,7 +48,7 @@
 			"02"
 			{
 				"var_type"					"FIELD_INTEGER"
-				"omni_slash_jumps"			"3 6 9 14 32"
+				"omni_slash_jumps"			"3 6 9 12 18"
```

Comment:
Honestly I don't think its' gonna hurt him too much. Omnislash is more about the initiation, the attack damage, and the invulnerability window. Having it last longer than it needs to is just overkill.